### DISCUSSION TITLE
How should I define my CI.yml for the social cards to work ?

### DISCUSSION THREAD

**Discussion by mathisgauthey** [squidfunk/mkdocs-material#6751] [Q&A] [answered]
Hey there lads, I configured all my website the way I wanted, it was really smooth sailing, the docs are pretty complete. Well, for a documentation site builder, it would be an issue otherwise.

Anyway, I just struggle to make the social cards working on my github pages. Is there anything I overlooked in my `ci.yml` ?

You can find it [here](https://github.com/mathisgauthey/mathisgauthey.github.io/blob/main/.github/workflows/ci.yml), my website source being public.

Thanks in advance, I'm still searching and will post here if I found a fix.

**Comment by kamilkrzyskow** [comment, marked as answer]
Hello @mathisgauthey,
the answer requires investigation, but there were multiple similar questions to this one concerning issues with the social cards, so you should be able to find them and reference. So please make sure to search for discussions in the future.

As for your specific site, the social cards are generated on the deployed branch: 
https://github.com/mathisgauthey/mathisgauthey.github.io/tree/gh-pages/assets/images/social

However, checking on https://developers.facebook.com/tools/debug/ or https://www.linkedin.com/post-inspector/ the images aren't loading, it's worth noting sometimes there are issues with cache involved, causing a delay, but this time the site was loaded from scratch, so it should work.

Therefore I had a look in the HTML `<head>` and it turned out that the link to the social cards is invalid:
https://mathis.gauthey.github.io/assets/images/social/index.png and your domain is https://mathisgauthey.github.io/

So fix your `site_url` in the `mkdocs.yml` https://github.com/mathisgauthey/mathisgauthey.github.io/blob/main/mkdocs.yml#L4

**Reply by mathisgauthey** [reply]
I just found out about the built social cards in my gh-pages branch when I received your answer.

Thanks for including the way you figured things out, I learned new things there. I was using discord to see if my social cards were working, but these post inspectors sure do good as well.

I never thought about looking at the HTML `<head>`, still got plenty to learn.

Anyway, thanks, it's fixed now !
